packages/flutter/test/gestures/debug_test.dart

```diff
@@ -25,7 +25,7 @@ void main() {
     tap.addPointer(event);
     expect(log, hasLength(2));
     expect(log[0], equalsIgnoringHashCodes('Gesture arena 1    ❙ ★ Opening new gesture arena.'));
-    expect(log[1], equalsIgnoringHashCodes('Gesture arena 1    ❙ Adding: TapGestureRecognizer#00000(state: ready)'));
+    expect(log[1], equalsIgnoringHashCodes('Gesture arena 1    ❙ Adding: tap(state: ready)'));
```

Comment:
argh, yeah, that's no good. We need the hashcode here so we can recognize it in the logs.

Ok, how about for the recognizers we have an entirely separate "debugDescription" getter that's totally independent of the diagnostics stuff, which we use for the short name, and anywhere where previously we used toStringShort, we now use that?